linera-core/src/unit_tests/client_tests.rs

```diff
@@ -1599,8 +1682,23 @@ where
     client0.process_inbox().await.unwrap();
     assert_eq!(
         client0.local_balance().await.unwrap(),
-        Amount::from_tokens(7)
+        Amount::from_tokens(9)
     );
+    assert!(client0.pending_block.is_none());
+
+    // Burn another token so Client 1 sees that the blob is already published
+    client1
+        .burn(None, Amount::from_tokens(1), UserData::default())
+        .await
+        .unwrap();
+    client1.synchronize_from_validators().await.unwrap();
```

Comment:
Shouldn't this be enough to see that it's published? No need to burn another precious token!

Reply:
Synchronizing should be enough, but I wanted to make sure by checking the pending block/blobs, and that'll only get cleared if I do another operation :/